app/src/main/res/values/strings.xml

```diff
@@ -98,6 +98,7 @@
   <string name="err_invalid_phone">Please enter a valid phone number</string>
   <string name="err_address_blank">Please enter an address</string>
   <string name="err_cant_load_photo">Unable to load photo</string>
+  <string name="err_cant_load_photo_bad_extension">Sorry\, we couldn\'t load your file because it isn\'t an image.</string>
```

Comment:
This is better but it's possible to choose a valid image and get this error message.  Can we just make it say "Sorry, unable to load you image"